### PR TITLE
Update Deutsch locale

### PR DESCRIPTION
**Pull request by TooWaBoo**
ping @heckflosse :-)

### CODE REVIEW

File: rtdata/languages/Deutsch [hunk 1]
```diff
@@ -1832,9 +1833,9 @@ TP_RAWEXPOS_LINEAR;Korrekturfaktor
 TP_RAWEXPOS_PRESER;Weißpunkt: Lichter schützen
 TP_RAWEXPOS_RGB;Rot, Grün, Blau
 TP_RAWEXPOS_TWOGREEN;Grün-Werte automatisch angleichen
-TP_RAW_1PASSMEDIUM;1-Pass (Medium)
-TP_RAW_3PASSBEST;3-Pass (Beste)
-TP_RAW_4PASS;4-Pass
+TP_RAW_1PASSMEDIUM;1-Pass (Markesteijn)
+TP_RAW_3PASSBEST;3-Pass (Markesteijn)
```
Comment (ff2000) [inline rtdata/languages/Deutsch:1837]: Seems the above were reverted to strings from default language file. Is that intentional?
[And I had to google Markesteijn - seems to be an algorithm, is that correct?]

Comment (TooWaBoo) [reply rtdata/languages/Deutsch:1837]: Yes, it's all correct and tested with latest "dev". @heckflosse changed it yesterday. There is also a discussion at pixel.us about that.
https://discuss.pixls.us/t/my-sharpening-workflow-for-base-iso-mages-with-the-new-features-in-rt-aka-extreme-pixel-peeping/8034/68

**Comment by TooWaBoo** [comment]
ping @heckflosse :-)

**Comment by heckflosse** [comment]
@TooWaBoo Thanks for the ping :+1:

**Pull request merged**